app/controllers/catalog_controller.rb

```diff
@@ -307,6 +307,10 @@ def collection_search
 
     # sort by title
     params[:sort] = 'title_ssi asc' unless params[:sort]
+
+    # RDCP prefers their collections sorted by most recently created first
+    # Therefore, we're sorting by modified date rather than title
+    params[:sort] = 'system_modified_dtsi desc, title_ssi asc' if params[:id].eql? 'rdcp'
```

Comment:
My thinking was that this would be last in the chain of sorting checks. If I put it above, then it could potentially get overwritten later. But I may be misunderstanding you, can you help clarify with an example maybe?

Reply:
@mcritchlow I think it's just related to your last comment in https://github.com/ucsdlib/damspas/pull/663#discussion_r295917873. The default `sort by title` will be set with the `unless params[:sort]` in https://github.com/ucsdlib/damspas/pull/663/files#diff-956e46b33fb5307990c4e1a4b5fd86ccR309 only when there are no sort order rules are selected.